yogstation/code/game/objects/items/devices/radio/headset.dm

```diff
@@ -3,4 +3,11 @@
 	desc = "A headset that is a result of the mating between medical and cargo. \nTo access the medical channel, use :m. For cargo, use :u."
 	icon_state = "med_headset"
 	item_state = "headset"
-	keyslot = new /obj/item/encryptionkey/headset_medsup
\ No newline at end of file
+	keyslot = new /obj/item/encryptionkey/headset_medsup
+
+/obj/item/radio/headset/headset_medsec
+	name = "mining medic radio headset"
```

Comment:
this doesn't seem right

Reply:
copied from the mining medic's headset, forgot to change it